.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,23 @@
+{
+   "configurations": [
+      {
+         "name": "Win32",
+         "includePath": [
+            "${workspaceFolder}/**",
+            "${QTDIR}/include/**"
+         ],
+         "defines": [
+            "_DEBUG",
+            "UNICODE",
+            "_UNICODE"
+         ],
+         "windowsSdkVersion": "10.0.22621.0",
```

Comment:
Do we need it hardcoded here? I believe that it can be detected automatically.

Reply:
As far as I can see, this is (c_cpp_properties.json) an automatically created file, it is better to add it to gitignore.

Reply:
> Do we need it hardcoded here? I believe that it can be detected automatically.

Probably.

Reply:
> As far as I can see, this is (c_cpp_properties.json) an automatically created file, it is better to add it to gitignore.

The `includePath`, adding the QTDIR, isn't default and it's something I enjoy.
Once the file is created, I'm not sure it gets modified. I suggest we give it a try and re-assess if we see that it does and it becomes a nuisance.

Reply:
I believe we can add Qt to the include path in our cmake files.
Although it's strange that I didn't need it in my example...
Maybe because I have Qt in the PATH